shared/js/background/events.es6.js

```diff
@@ -33,7 +33,21 @@ function getFloat () {
     return crypto.getRandomValues(new Uint32Array(1))[0] / 2 ** 32
 }
 
-function getHash () {
+async function getHash () {
+    let hash = await browserWrapper.getFromSessionStorage('sessionKey')
+    if (!hash) {
+        hash = setNewHash()
```

Comment:
I'd prefer to make `setNewHash` `async`, have it `await` on `browserWrapper.setToSessionStorage` and then `await` on the `setNewHash` call here. I don't think a race condition (where we call `getHash` again before finished writing the hash to storage) is likely though to be fair.